content/influxdb/cloud-dedicated/reference/sql/explain.md

```diff
@@ -39,36 +49,75 @@ GROUP BY room
+|    |               |<span style="white-space:pre-wrap;">       RepartitionExec: partitioning=Hash([room@0], 8), input_partitions=8                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       </span>|
+|    |               |<span style="white-space:pre-wrap;">         AggregateExec: mode=Partial, gby=[room@0 as room], aggr=[AVG(home.temp)]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                </span>|
+|    |               |<span style="white-space:pre-wrap;">           ParquetExec: file_groups={8 groups: [[70434/116281/404d73cea0236530ea94f5470701eb814a8f0565c0e4bef5a2d2e33dfbfc3567/1be334e8-0af8-00da-2615-f67cd4be90f7.parquet, 70434/116281/b7a9e7c57fbfc3bba9427e4b3e35c89e001e2e618b0c7eb9feb4d50a3932f4db/d29370d4-262f-0d32-2459-fe7b099f682f.parquet], [70434/116281/c14418ba28a22a3abb693a1cb326a63b62dc611aec58c9bed438fdafd3bc5882/8b29ae98-761f-0550-2fe4-ee77503658e9.parquet], [70434/116281/fa677477eed622ae8123da1251aa7c351f801e2ee2f0bc28c0fe3002a30b3563/65bb4dc3-04e1-0e02-107a-90cee83c51b0.parquet], [70434/116281/db162bdd30261019960dd70da182e6ebd270284569ecfb5deffea7e65baa0df9/2505e079-67c5-06d9-3ede-89aca542dd18.parquet], [70434/116281/0c025dcccae8691f5fd70b0f131eea4ca6fafb95a02f90a3dc7bb015efd3ab4f/3f3e44c3-b71e-0ca4-3dc7-8b2f75b9ff86.parquet], ...]}, projection=[room, temp] </span>|
```

Comment:
@sanderson I'm undecided if this is more readable than using a code block. I'm going to leave them in `explain.md` files for now. If this is preferable then I assume we'll want to create a shortcode to style the `td` for query plan examples and then remove these `span`.